packages/@textlint/fixer-formatter/test/formatters/raw-output-test.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+import path from "path";
+import rawOutput from "../../src/formatters/raw-output";
+import assert from "assert";
+
+const formatter = (code) => {
+    return rawOutput(code, { color: false });
+};
+
+describe("formatter:raw-output", function () {
+    context("when single modified", function () {
+        it("should return output", function () {
+            const input = path.join(__dirname, "../fixtures", "single.md");
+            const code = require("../fixtures/single");
+            const output = formatter(code, { color: false });
+            assert.equal(
+                output,
+                `
+1st line
+2nd line
+3rd line
+4th line
+5th line
+6th line
+7th line
+`
+            );
+        });
+    });
+    context("when double modified", function () {
+        it("should return output", function () {
+            const input = path.join(__dirname, "../fixtures", "double.md");
+            const code = require("../fixtures/double");
+            const output = formatter(code, { color: false });
+            assert.equal(
```

Comment:
```suggestion
            assert.strictEqual(
```

Can you use strictEqual?

Reply:
Fixed in https://github.com/textlint/textlint/commit/0776215a49d47fbe050a11dea80d5b2780381a9a